source/network-security.txt

```diff
@@ -12,18 +12,137 @@ Network Security
    :depth: 1
    :class: onecol
 
-Intro statement
+By default, {+service+} secures your database deployments through the
+use of mandatory TLS/SSL connection encryption, and a VPC-first,
+minimum-access default policy. You can further configure these
+protections to meet your unique security needs and preferences.
 
-{+service+} Features and Best Practices for Network Security
-------------------------------------------------------------
+Use the foundational guidance on this page to plan the network
+security configuration for your {+clusters+}.
 
-Content here
+{+service+} Features and Recommendations for Network Security
+-------------------------------------------------------------
+
+Features
+~~~~~~~~
+
+{+service+} enforces TLS/SSL encryption for all connections to your
+databases.
+
+All {+service+} projects with one or more M10+ dedicated {+clusters+}
+receive their own dedicated VPC (or VNet if you use
+|azure|). {+service+} deploys all dedicated clusters inside this VPC
+or VNet. By default, this VPC or VNet allows no inbound access to
+{+service+}, requiring you to explicitly enable access by one of the
+following methods:
+
+- Add public IP addresses to your IP access list
+- Use VPC / VNet peering to add private IP addresses
+- Add private endpoints
+
+TLS
+```
+
+{+service+} enforces mandatory TLS encryption of connections to your
+databases. TLS 1.2 is the default protocol; you can select TLS 1.1 or
+TLS 1.0 if necessary. Note that in situations where TLS 1.1 or later
+is available, MongoDB 4.0 or later doesn't support TLS 1.0.
+
+IP Access Lists
+```````````````
+
+You can configure IP Access Lists to limit which IP addresses can
+attempt authentication to your database. Unless you have added the
+relevant IP addresses or CIDR blocks covering the relevant IP
+addresses to your access list, application servers and other clients
+cannot access your {+service+} {+clusters+}.
+
+You can configure temporary access list entries that expire
+automatically after a user-defined period.
+
+You can create one access list per project.
+
+Firewall Configuration
+``````````````````````
+
+If you use a firewall that blocks outbound network connections, you
+must also configure your firewall to allow your applications to make
+outbound connections to TCP traffic on {+service+} hosts. This grants
+your applications access to databases stored on {+service+}.
+
+{+service+} {+cluster+} public IPs remain the same in the majority of
+cases of {+cluster+} changes: vertical scaling, topology changes,
+maintenance events, healing events, etc. However, certain topology
```

Comment:
remove etc. What is vertical scaling in this case? What are topology changes? What are maintenance or healing events? Please try to provide simple examples or replace these words with specific scenarios.